SU-SWS/stanford_profile
# D8CORE-2538 Staff, Faculty, and Students shouldnt see the admin toolbar

# READY FOR REVIEW

# Summary
- Removed permission for anonymous, staff, students, and faculty from using the admin toolbar or seeing the admin theme.

# Need Review By (Date)
- 10/1

# Urgency
- medium

# Steps to Test
1. view the tests or
1. checkout this branch
1. install/drush cim
1. view permissions page and verify the toolbar and admin theme are only enabled for the correct roles.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
